common/scala/src/main/resources/kafka.conf

```diff
@@ -0,0 +1,24 @@
+kafka {
```

Comment:
should the corresponding values in the ansible group vars be removed?

Reply:
yep.  good catch.  done.

Reply:
removed.  I thought it was still needed because WhiskConfig is mentioned in the constructor down below, but I guess I don't understand Scala's import rules...only have to import if you actually do something with the type (not just passing it through?)

Reply:
But only if those are valid for **each** environment?